tools/build_pytorch_libs.py

```diff
@@ -118,10 +118,8 @@ def create_build_env():
         # But it should be set to MSVC as the user's first choice.
         if USE_NINJA:
```

Comment:
nit: What about merging the `if`s here too?

Reply:
Done!